tools/fileinfo/features/imports-exports/test.py

```diff
@@ -175,3 +175,18 @@ def test_correctly_analyzes_imports_and_exports(self):
         self.assertEqual(self.fileinfo.output['exportTable']['exports'][1]['address'], '0x12990')
         self.assertEqual(self.fileinfo.output['exportTable']['exports'][2]['address'], '0x12994')
         self.assertEqual(self.fileinfo.output['exportTable']['exports'][3]['address'], '0x12998')
+
+class ExportByOrdinalTest(Test):
+    settings = TestSettings(
+        tool='fileinfo',
+        input='export_ordinals.dll',
+        args='--verbose --json'
+    )
+
+    def test_correctly_analyzes_imports_and_exports(self):
```

Comment:
Shouldn't the name of the test be just `test_correctly_analyzes_exports`? There does not seem to be anything dealing with imports in the test.